must-gather/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM quay.io/openshift/origin-cli:latest
```

Comment:
@bparees do you know if there is a way for us to pull `oc` into the CLO image and maybe provide it as part of CLO?  @periklis can we provide an alternate entrypoint with must gather

The goal here is to not need to productize yet another image

Reply:
AFAICS, other operators just publish their must-gather images on developer repos on quay.io, e.g.:
https://github.com/openshift/ocs-operator/tree/master/must-gather#usage
https://github.com/openshift/ocs-operator/tree/master/must-gather#usage

You can do the same w/o productization, can't we?

Reply:
The discussion we had with thin the arch group was to include this as part of the CLO.  Suggestion was to base the CLO image on `quay.io/openshift/origin-cli:latest` and find a way to invoke an alternate entrypoint or use whatever convention that is provided for must gather.  Is there any reason NOT to do this? @alanconway

Reply:
@jcantrill This is exactly what we have right now. Or do you see any difference.

Reply:
>  Suggestion was to base the CLO image on quay.io/openshift/origin-cli:latest

it is different than what it suggested here in that we wouldn't have a second image instead it would be invoked from the CLO image rather than this `must-gather` one

Reply:
I adapted the packaging to add everything in the single operator image. You can try it like this:
```
oc adm must-gather --image=quay.io/periklis/cluster-logging-operator:latest -- /usr/bin/gather
```
WDYT? (cc @jcantrill @ewolinetz)

Reply:
my only concern with that is we would become an outlier on how to invoke `oc adm must-gather` vs how other teams would be called... 

unless there are other teams out there that would have do something similar, i would rather it be invoked like:
```
oc adm must-gather --image=quay.io/periklis/cluster-logging-operator:latest
```

and then we update how we invoke the normal operator process within our deployment (the entrypoint can be specified in that definition)

Reply:
That's the point, you can not define the entrypoint via `oc adm must-gather`. You can only override the cmd via `-- /usr/bin/gather`. I believe this is ok, because user will just copy and paste the full oc-adm command from docs.

Reply:
I believe the fact it is an outlier is insignificant as long as we document it.  Documentation is significantly easier then having to setup the release train.  Also, users then have the scripts instead of having to go and find the image